lib/id3c/cli/command/manifest.py

```diff
@@ -445,3 +542,10 @@ def load_ndjson(file: Iterable[str]) -> Iterable:
     Load newline-delimited JSON records from *file*.
     """
     yield from (json.loads(line) for line in file)
+
+
+def find_reserved_columns_used_in_extra_column_list(extra_columns: List[Tuple[str, Union[str, dict]]]):
+    """
+    Find any reserved column names that also appeaer in extra_columns.
+    """
+    return [x for x in extra_columns if str(x[0]).lower() in RESERVED_COLUMNS]
```

Comment:
This function name feels pretty verbose and specific to me, particularly for an internal function which can benefit from the caller's context being in the same file.

I wonder if this is better expressed in set logic:

```
disallowed_extra_columns = {dst for dst, src in extra_columns} & RESERVED_COLUMNS
```

Is the string casting so the config can do `42: …` instead of `"42": …` in `extra_columns`?

Why the `lower()`? We're not doing case-insensitive matching on this later down the line, are we?

Reply:
Python sets functionality is really cool. Thinking of this as the intersection of two sets is so great. I've made the switch.
Also, we don't to lower() anywhere else in here. That was me thinking of case insensitive SQL instead of JSON.